test/test_grade/test_integration.py

```diff
@@ -312,6 +343,7 @@ def test_config_overrides_integration():
         "percent_correct": float('nan'),
         "total_points_earned": 13.0,
         "file": POINTS_POSSIBLE_LABEL,
+        "grading_status": float('nan')
```

Comment:
Why is this nan? the grading status should be a string

Reply:
The "NA" gets coerced by pandas' read_csv in the testing. I am suggesting for `grading_status` on the points-per-question row we use the string "--".